hatch.toml

```diff
@@ -9,3 +9,19 @@ dependencies = ["setuptools"]  # https://bitbucket.org/pybtex-devs/pybtex/issues
 [envs.docs.scripts]
 build = "sphinx-build -M html docs docs/_build -W --keep-going {args}"
 clean = "git clean -fX -- docs"
+
+[envs.hatch-test]
```

Comment:
Why not use `uv` by default? Is this inherited from `default`?

Reply:
- all envs *except* the `hatch-*` ones inherit from default, that’s why this works without `envs.towncrier.features` being defined: https://github.com/scverse/anndata/blob/94b2304cb2c08dd2c5f8d3cb7ee4da39d874975a/hatch.toml#L13-L15
- the `hatch-*` envs have `uv` by default